src/Compositor.cpp

```diff
@@ -1278,9 +1292,9 @@ void CCompositor::moveWindowToTop(CWindow* pWindow) {
     }
 
     for (auto& pw : toMove) {
-        moveToTop(pw);
+        moveToZ(pw, top);
 
-        moveWindowToTop(pw);
+        changeWindowZOrder(pw, top);
```

Comment:
both of these should be true, not top

Reply:
Nope, it's uglier than that... 
For the X11 windows I'll need to figure out what order toMove has the windows in and then move them to the bottom Z from the top of the stack down.   I'm thinking the original is just doing a 'top' form the bottom of the stack up.  It'll take me more than the time I have tonight to fix that and test it.

Reply:
parent -> child -> child


you move the parent to the bottom, but children should still be preserved on top of it.

You're right it shouldn't be `true`, my bad. it should be children from the bottom first to the bottom, parent last, lol